lib/models/validationErrors.js

```diff
@@ -0,0 +1,76 @@
+module.exports = {
+    default: {
```

Comment:
I'd add for each field an error attribute this way it's consistent with how the rest of our errors are returned to users